buildSrc/src/main/groovy/com/github/okbuilds/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -133,6 +133,18 @@ final class BuckFileGenerator {
             ":${rule.name}"
         }
 
+        // SqlDelight
+        if (target.sqldelight != null) {
+            String runner = target.sqldelight.externalDeps.find { it.contains("SqlDelightBin") }
+            if (runner == null) {
+                throw new IllegalStateException("SqlDelight model module must declare the dependency: " +
+                        "`provided(group: 'com.github.okbuilds', name: 'SqlDelightBin', version: '1.0.0', ext: 'jar', classifier: '')`")
```

Comment:
Could we add dependency into the project after resolve?

If not, I think we should't add the SqlDelightBin dependency arbitrarily, it should only be added when user really use SqlDelight.

I'll change the error message to refer to our wiki page.